### PR TITLE
Make sure cluster tests actually created a cluster before running

### PR DESCRIPTION
**Pull request by jwilder**

### CODE REVIEW

File: cmd/influxd/server_integration_test.go [hunk 1]
```diff
@@ -157,6 +158,18 @@ func createCombinedNodeCluster(t *testing.T, testName, tmpDir string, nNodes int
 
 	}
 
+	// Sanity check that we created a cluster and data nodes have unique ids
+	ids := []int{}
```
Comment (otoolep) [inline cmd/influxd/server_integration_test.go]: Would it be clearer if you just did something like this:
```
ids := make(map[int]bool)
for _, n := range nodes {
    ids[int(n.node.DataNode.ID())] = true
}
if len(ids) != len(nodes) {
    t.Fatalf("failed to create valid cluster")
}
```
No sorting required.

**Comment by otoolep** [comment]
Worthwhile test.

**Comment by otoolep** [comment]
I took at look at the diagnostics dump from each node, it would be worth examining in greater detail to see what has happened to the node. The dumps do confirm that each node has a distinct ID.

**Comment by otoolep** [comment]
+1

**Comment by toddboom** [comment]
:+1:

**Pull request merged**